src/bindings/python/src/openvino/runtime/ie_api.py

```diff
@@ -37,8 +37,14 @@ def normalize_inputs(py_dict: dict, py_types: dict) -> dict:
 
 
 def get_input_types(obj: Union[InferRequestBase, ExecutableNetworkBase]) -> dict:
```

Comment:
@jiwaszki @akuporos please have a look at this function. I've adjusted it to handle corner cases like a model with a single Identity operator which gets reduced to Parameter->Result. In such cases the tensor in between has multiple names instead of one.